unison-src/transcripts/branch-command.output.md

```diff
@@ -30,85 +30,85 @@ foo/main> branch topic1
 
   Done. I've created the topic1 branch based off of main.
   
-  Tip: Use `merge /topic1 /main` to merge your work back into
-       the main branch.
+  Tip: To merge your work back into the main branch, first
+       `switch /main` then `merge /topic1`.
 
 foo/main> branch /topic2
 
   Done. I've created the topic2 branch based off of main.
```

Comment:
unrelated issue, but maybe we want to use the slash syntax here too?

Reply:
I don't think so; the slash isn't part of the branch name. Maybe that's also an argument against using it in the "I merged /bob into /alice" output message 🤔